classes/pdf/HTMLTemplateInvoice.php

```diff
@@ -153,8 +153,8 @@ public function getContent()
                     $orderDetail['total_price_tax_excl_including_ecotax'] = $orderDetail['unit_price_tax_excl_including_ecotax'] * $orderDetail['product_quantity'];
                     break;
                 case Order::ROUND_LINE:
-                    $orderDetail['total_price_tax_excl'] = Tools::ps_round($orderDetail['total_price_tax_excl'], _PS_PRICE_DISPLAY_PRECISION_);
-                    $orderDetail['total_price_tax_excl_including_ecotax'] = Tools::ps_round($orderDetail['unit_price_tax_excl_including_ecotax'], _PS_PRICE_DISPLAY_PRECISION_);
+                    $orderDetail['total_price_tax_excl'] = Tools::ps_round(($orderDetail['total_price_tax_excl'] * $orderDetail['product_quantity']), _PS_PRICE_DISPLAY_PRECISION_);
```

Comment:
I believe it's okay with the quantity removed from this line. `total_price_tax_excl` should always be the line total and therefore does not need a quantity.